thirdparty/mbedtls/include/godot_module_mbedtls_config.h

```diff
@@ -41,18 +41,27 @@
 #else
 
 // Include default mbedTLS config.
-#include <mbedtls/config.h>
+#include <mbedtls/mbedtls_config.h>
 
 // Disable weak cryptography.
 #undef MBEDTLS_KEY_EXCHANGE_DHE_PSK_ENABLED
 #undef MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED
-#undef MBEDTLS_SSL_CBC_RECORD_SPLITTING
-#undef MBEDTLS_SSL_PROTO_TLS1
-#undef MBEDTLS_SSL_PROTO_TLS1_1
-#undef MBEDTLS_ARC4_C
 #undef MBEDTLS_DES_C
 #undef MBEDTLS_DHM_C
 
+// Disable TLSv1.3
+#undef MBEDTLS_SSL_PROTO_TLS1_3
+
+// Disable PSA Crypto
```

Comment:
Would be good to add comments explaining why we're disabling those (especially as IIUC you intend to enable them in the future, so this is temporary).
```suggestion
// Disable TLSv1.3.
#undef MBEDTLS_SSL_PROTO_TLS1_3

// Disable PSA Crypto.
```

Reply:
Added a comment explaining the reasons.